### PR TITLE
Consistency fixes

### PR DESCRIPTION
**Pull request by rbuzatu90**

### CODE REVIEW

File: jobs/run_collect.sh [hunk 1]
```diff
@@ -79,18 +80,6 @@ if [ "$IS_DEBUG_JOB" != "yes" ]
                 echo "Removing HyperV temporary console logs.."
                 rm -fv /home/jenkins-slave/logs/hyperv-build-log-$ZUUL_UUID-$hyperv01
                 rm -fv /home/jenkins-slave/logs/hyperv-build-log-$ZUUL_UUID-$hyperv02
-
-		echo "Releasing devstack floating IP"
-		nova remove-floating-ip "$VMID" "$FLOATING_IP"
```
Comment (ociuhandu) [inline jobs/run_collect.sh:84]: why is this removed?

Comment (rbuzatu90) [inline jobs/run_collect.sh:84]: Because with the new workflow we cleanup in a separate script, aka run_cleanup.sh
